telegram/ext/filters.py

```diff
@@ -191,7 +192,7 @@ class MessageFilter(BaseFilter):
 
     __slots__ = ()
 
-    def check_update(self, update: Update) -> Optional[Union[bool, DataDict]]:
+    def check_update(self, update: Update) -> Optional[Union[bool, DataDict]]:  # type: ignore
```

Comment:
if the `type: ignore` is due to `BaseFilter.check_update` being static: I'd vote to rather ignore the pylint error for not using `self` in `BaseFilter.check_update`.